commands/middleware/middleware.go

```diff
@@ -8,11 +8,12 @@ import (
 	"github.com/Sirupsen/logrus"
 	logContext "github.com/docker/distribution/context"
 	strfmt "github.com/go-openapi/strfmt"
+	authContext "github.com/netlify/open-api/go/porcelain/context"
```

Comment:
It will respect whitepspace and cluster them :) so I do it in 4 stanzas really:

- sytem
- everyone else
- libs we control
- this tool

b/c the import tool screws up and imports the wrong lib ~ how many do we have called 'conf' and then it is a "nightmare" to find.